docs/io/unified_table_text.rst

```diff
@@ -16,6 +16,12 @@ powerful interface for reading and writing text tables. In addition, the interfa
 provides wrappers around select I/O functions in the `pandas`_ library for additional
 flexibility and performance.
 
+.. note::
+
+   For reading large CSV files, the astropy :ref:`PyArrow CSV <table_io_pyarrow_csv>`
+   reader is a good option to consider since it can be up to 15 times faster than other
```

Comment:
Seeing it again, I think "is recommended" is the way to go here!

Reply:
On this matter I think there are many valid considerations that might influence the decision each user makes. Maybe they don't want to install pyarrow, or maybe `io.ascii` is already working well enough, or maybe they use some feature of the pandas reader. So here the goal is to call attention to this option without making a ranked value judgement.